magma/primitives/mux.py

```diff
@@ -108,7 +107,7 @@ def _infer_mux_type(args):
             if issubclass(T, next_T):
                 # upcast
                 T = next_T
-            elif not wireable(next_T, T):
```

Comment:
can we get rid of `magma/wireable.py` then?

Reply:
Already removed by https://github.com/phanrahan/magma/pull/892/commits/69a2783b0110e230261c2d6d0af4005b9f9d762a